tests/testthat/helper-integration-tests-setup-function.R

```diff
@@ -542,55 +542,18 @@ setup_and_run_FIMS_with_wrappers <- function(iter_id,
   clear()
 
   data <- FIMS::FIMSFrame(data1)
-
-  # Set up default parameters
-  fleets <- list(
-    fleet1 = list(
-      selectivity = list(form = "LogisticSelectivity"),
-      data_distribution = c(
-        Landings = "DlnormDistribution",
-        Index = "DlnormDistribution",
-        AgeComp = "DmultinomDistribution",
-        LengthComp = "DmultinomDistribution"
-      )
-    ),
-    survey1 = list(
-      selectivity = list(form = "LogisticSelectivity"),
-      data_distribution = c(
-        Landings = "DlnormDistribution",
-        Index = "DlnormDistribution",
-        AgeComp = "DmultinomDistribution",
-        LengthComp = "DmultinomDistribution"
-      )
-    )
-  )
-
-  default_parameters <- data |>
-    create_default_parameters(
-      fleets = fleets,
-      recruitment = list(
-        form = "BevertonHoltRecruitment",
-        process_distribution = c(log_devs = "DnormDistribution")
-      ),
-      growth = list(form = "EWAAgrowth"),
-      maturity = list(form = "LogisticMaturity")
-    )
-
-  parameters <- default_parameters |>
-    update_parameters(
-      modified_parameters = modified_parameters[[iter_id]]
-    )
+  parameters <- modified_parameters
 
   # The model will not always run when log_q is very small.
   # We will need to make sure log_q is the true value for deterministic runs but
   # then reset to log(1.0) for estimation runs.
-  if (estimation_mode == TRUE) {
+  if (estimation_mode == TRUE){
```

Comment:
There should be a space here so the change is not needed, might run styler on the changes?

Reply:
Good catch. Done!